core/src/Config.purs

```diff
@@ -481,7 +481,13 @@ readConfig :: forall a. FilePath -> Spago (LogEnv a) (Either String { doc :: Yam
 readConfig path = do
   logDebug $ "Reading config from " <> path
   FS.exists path >>= case _ of
-    false -> pure $ Left $ case path of
-      "spago.yaml" -> "Did not find " <> path <> " Run `spago init` to initialise a new project."
-      _ -> "Did not find " <> path
+    false -> case path of
+      "spago.yaml" -> do
+        hasYml <- FS.exists "spago.yml"
```

Comment:
It would be better to search for `spago.yml` in the same directory where the config file (`path`) is located.

Reply:
Indeed, we use this function also when looking for workspace packages, where it's possible that the user has used the `.yml` extension.

I would use [`String.stripSuffix`](https://pursuit.purescript.org/packages/purescript-strings/6.0.1/docs/Data.String#v:stripSuffix) to detect if the config we're looking for has a `.yml` extension, instead of matching only on the root config